x-pack/solutions/security/plugins/security_solution/public/detection_engine/rule_management/components/rule_details/base_version_diff/base_version_flyout.tsx

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo, useMemo } from 'react';
+import { EuiButton, EuiCallOut, EuiSpacer, EuiToolTip } from '@elastic/eui';
+import type { PartialRuleDiff, RuleResponse } from '../../../../../../common/api/detection_engine';
+import { PerFieldRuleDiffTab } from '../per_field_rule_diff_tab';
+import { RuleDetailsFlyout, TabContentPadding } from '../rule_details_flyout';
+import * as ruleDetailsI18n from '../translations';
+import * as i18n from './translations';
+import { RuleDiffTab } from '../rule_diff_tab';
+import { BaseVersionDiffFlyoutSubheader } from './base_version_flyout_subheader';
+import { useRevertPrebuiltRule } from '../../../logic/prebuilt_rules/use_revert_prebuilt_rule';
+
+export const PREBUILT_RULE_BASE_VERSION_FLYOUT_ANCHOR = 'baseVersionPrebuiltRulePreview';
+
+interface PrebuiltRulesBaseVersionFlyoutComponentProps {
+  currentRule: RuleResponse;
+  baseRule: RuleResponse;
+  diff: PartialRuleDiff;
+  closeFlyout: () => void;
+  isReverting: boolean;
+  onRevert?: () => void;
+}
+
+const PrebuiltRulesBaseVersionFlyoutComponent = ({
```

Comment:
nit: This is a shorter form

```suggestion
export const PrebuiltRulesBaseVersionFlyout = memo(function PrebuiltRulesBaseVersionFlyout({
  currentRule,
  baseRule,
  diff,
  closeFlyout,
  isReverting,
  onRevert,
}: PrebuiltRulesBaseVersionFlyoutComponentProps): JSX.Element {
  ...
}
```